synapse/handlers/appservice.py

```diff
@@ -226,7 +226,7 @@ async def _notify_interested_services_ephemeral(
         new_token: Optional[int],
         users: Collection[Union[str, UserID]],
     ):
-        logger.info("Checking interested services for %s" % (stream_key))
+        logger.debug("Checking interested services for %s" % (stream_key))
```

Comment:
Slightly worried that we're saying "services" here whereas everywhere else we use "appservices". A fix for another PR though.

Reply:
I believe this is fine, as the logs are scoped to "appservice" anyway, so you're unlikely to be confused by it. Short-handing it seems fine with that context?

Reply:
I suppose it will print the file that this line is coming from in the log. I was just slightly concerned that people will have to wonder what "service" the line is talking about here.

It was mostly a concern over confusing wording when parsing the logs. When developing this being inside of `ApplicationServicesHandler` is pretty obvious yeah.

Reply:
I thought that the logs prefix'd the module so you won't need to oversupply your own context, but I might be misremembering.